### PR TITLE
Update intro.md

### PR DESCRIPTION
**Pull request by tristan-spaulding2**
## Description

Rewrites intro match marketing message more tightly and tie to value points.

Moves Feature flag section lower so that a user can more quickly get a sense of what's happening.

## Ticket
Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist
Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply. 

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Review by KyleGoyette — APPROVED** [review]

File: docs/guides/launch/intro.md [hunk 1]
```diff
@@ -3,27 +3,14 @@ slug: /guides/launch
...
-
-1. Navigate to https://wandb.ai/settings.
-2. Scroll down to the **Beta Features** section and enable **W&B Launch**.
-
-![](/images/launch/toggle_beta_flag.png)
 
 ## What is Launch?
 
-W&B Launch is a toolkit for building collaborative, portable, and reproducible machine learning workflows.
+W&B Launch introduces a connective layer between machine learning practitioner and the high-scale, specialized hardware that powers modern machine learning workflows. Easily scale training from your desktop to GPUs such as when you run intensive model evaluation suites or transfer models for production inference.
```
Comment (ngrayluna) [inline docs/guides/launch/intro.md]: Changed this last sentence to make it less market-y

**Review by ngrayluna — APPROVED** [review]

File: docs/guides/launch/intro.md (hunk 1 above)
Comment (tristan-spaulding2) [reply docs/guides/launch/intro.md]: This edit changes the meaning..  Scaling up training, running model evaluation suites, and transferring for inference are three parallel activities, the latter two aren't instances of the first.

Comment (ngrayluna) [reply docs/guides/launch/intro.md]: We can def can change it. That was my quick attempt to adhere to styling: https://developers.google.com/style/highlights

Comment (tristan-spaulding2) [reply docs/guides/launch/intro.md]: updated also to fix grammar

**Pull request merged**